caab-service/src/main/java/uk/gov/laa/ccms/caab/api/controller/ApplicationController.java

```diff
@@ -31,7 +33,8 @@ public ResponseEntity<Void> createApplication(
           final String caabUserLoginId,
           final ApplicationDetail applicationDetail) {
 
-    Long id = applicationService.createApplication(caabUserLoginId, applicationDetail);
+    currentUserHolder.set(caabUserLoginId);
```

Comment:
Is this to be refactored into ControllerAdvice or something at a later date?

Reply:
I have a change in the next branch to sort this with controller advice